packages/pros/examples/imu.rs

```diff
@@ -8,11 +8,12 @@ use pros::prelude::*;
 #[derive(Default)]
 pub struct Robot;
 
-impl SyncRobot for Robot {
-    fn opcontrol(&mut self) -> pros::Result {
+#[async_trait]
```

Comment:
Once this pr has been fast-forwarded this will need to be removed because we no longer use the `async_trait` crate.